src/components/VNavigationDrawer/VNavigationDrawer.js

```diff
@@ -190,8 +190,14 @@ export default {
     }
   },
 
+  created () {
```

Comment:
All of these can go away https://github.com/vuetifyjs/vuetify/blob/d682de2ebaea68ab94b38790c3a360145f77cc85/src/components/VNavigationDrawer/VNavigationDrawer.js